libhif/hif-reldep-list-private.h

```diff
@@ -1,7 +1,5 @@
 /*
- * Copyright (C) 2012-2013 Red Hat, Inc.
- *
- * Licensed under the GNU Lesser General Public License Version 2.1
+ * Copyright © 2016  Igor Gnatenko <ignatenko@redhat.com>
```

Comment:
So I think technically it's fine. Maybe the #pragma once isn't a good idea, but the rest I'm happy with.